packages/core/src/endowmentsToolkit.js

```diff
@@ -589,3 +613,71 @@ function defaultCreateFunctionWrapper(sourceValue, unwrapTest, unwrapTo) {
  * @param {object} unwrapTo
  * @returns {(...args: any[]) => any}
  */
+
+/**
```

Comment:
I don't think it really matters, but I am curious why you wanted to extract all these typedefs from the function implementations in the first place.

Reply:
because I was struggling to export the whole thing. extraction can now be undone

Reply:
note: we ended up pairing on this